db/db.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/locations', { useNewUrlParser: true });
+const db = mongoose.connection;
+
+// eslint-disable-next-line no-console
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => {
+  // eslint-disable-next-line no-console
+  console.log('connected to the database!');
+});
+
+
+const reviewSchema = new mongoose.Schema({
+  ID: Number,
+  name: String,
+  date: Date,
+  avatar: String,
+  text: String,
+  'rating-accuracy': Number,
+  'rating-communication': Number,
+  'rating-cleanliness': Number,
+  'rating-location': Number,
+  'rating-check-in': Number,
+  'rating-value': Number,
+});
+
+const locationSchema = new mongoose.Schema({
+  locationID: String,
+  reviews: [reviewSchema],
```

Comment:
Not too familiar with schema relationships in MongoDB, does this set the `reviews` field to store ObjectID values from the referenced table?